### PR TITLE
D3ASIM-430 new component Diamond Shape

### PR DESCRIPTION
**Pull request by Lelith**
**Who**: @lelith

**What / Why**: New Atom for identifying elements of the grid configuration.

I m not sure about the icon usage here as its different from the current approach, but i think this use case is different than the others.
i m using the svgs as inline background image of the classes. I think it makes more sense to just show the icon based on the class-name instead of the need to have a switch in the javascript code. Also its easier to change the color and sizes based on the additional classes.

I used this website to generate the inline svg css file (but ignored the whole compatibility process so far) http://www.grumpicon.com/

storybook

### CODE REVIEW

**Comment by eleventhirtyfour** [comment]
as far as i understood from the design i think that element should behave like a button with an active state right? or am i wrong? we could wrap that element inside a button element but i guess we need different states .

**Comment by Lelith** [comment]
@eleventhirtyfour its just a style that can be applied to any kind of element.

**Comment by eleventhirtyfour** [comment]
ah sorry i didnt ment to close it , just wanted to comment

File: lib/components/Diamond/Diamond.scss [hunk 1]
```diff
@@ -0,0 +1,93 @@
+@import '../../styles/themify';
+@import '../../styles/variables';
+
+.diamond {
+  transform: rotate(45deg);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: (93px/4);
```
Comment (cfleschhut) [inline lib/components/Diamond/Diamond.scss]: @Lelith Just wondering: Where do the `93` pixels come from? (And why then divide them by 4?)

Comment (Lelith) [reply lib/components/Diamond/Diamond.scss]: ah that should be 90, and divided by 4 was me trying to figure out to increase the box-size for the amount of the edges that are turned and therefor overflow the box

**Comment by Lelith** [comment]
@cfleschhut good idea! added it to the pr description

**Pull request merged**